extensions/amp-story/1.0/amp-story.css

```diff
@@ -12,12 +12,20 @@
 /** Common */
 :root {
   --story-page-vw: 1vw !important;
-  --story-page-vh: 1vh !important;
+  --story-page-vh: var(--story-dvh, 1vh) !important;
+  --story-page-100vh: calc(var(--story-page-vh) * 100) !important;
   --story-page-vmin: min(var(--story-page-vw), var(--story-page-vh)) !important;
   --story-page-vmax: max(var(--story-page-vw), var(--story-page-vh)) !important;
   font-size: calc(2.5 * var(--story-page-vh, 8px));
 }
 
+@supports (height: 1dvh) {
+  :root {
+    --story-page-vh: 1dvh !important;
+    --story-page-100vh: 100dvh !important;
```

Comment:
This line is unnecessary since line 16 (`--story-page-100vh: calc(var(--story-page-vh) * 100) !important`) does this for us